functions-app/report-psychometrician/service/psychometrician-report.service.js

```diff
@@ -15,8 +15,18 @@ const psychometricianReportService = {}
  * @param logger - function logger
  * @return {Promise<void>}
  */
-psychometricianReportService.batchProduceCacheData = async function (batchIds, logger) {
-  const checks = await psychometricianDataService.sqlFindCompletedChecksByIds(batchIds)
+psychometricianReportService.batchProduceCacheData = async function batchProduceCacheData (batchIds, logger) {
```

Comment:
@jon-shipley looks good but could we have a test spec for that method too?

Reply:
i'd say this was optional at this point given that these changes are to support tracing of a bug.  Definitely a good catch though George, and should be addressed at some point.

Reply:
This service already has 92% coverage  😌

Reply:
@GeorgeChatzigiannis - batchProduceCacheData is an existing procedure that just got moved.